src/Microsoft.DotNet.Interactive/Server/Serializer.cs

```diff
@@ -23,10 +21,8 @@ static Serializer()
             NumberHandling = JsonNumberHandling.AllowReadingFromString | JsonNumberHandling.AllowNamedFloatingPointLiterals,
             Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
         };
-        JsonSerializerOptions.Converters.Add(new ByteArrayConverter());
```

Comment:
We need to keep this.  When a notebook is saved it's sent back as a byte array, but serialized as a base64 string.